daffodil-schematron/src/main/scala/org/apache/daffodil/validation/schematron/SchematronValidator.scala

```diff
@@ -17,40 +17,46 @@
 
 package org.apache.daffodil.validation.schematron
 
+import java.io.File
 import java.io.FileOutputStream
 import java.io.InputStream
-import java.nio.file.Path
+import java.net.URI
 import scala.util.Try
 import scala.xml.Elem
 import scala.xml.XML
 
-import org.apache.daffodil.lib.api.ValidationException
-import org.apache.daffodil.lib.api.ValidationFailure
-import org.apache.daffodil.lib.api.ValidationResult
-import org.apache.daffodil.lib.api.Validator
+import org.apache.daffodil.api
+import org.apache.daffodil.api.validation.Validator
 
 /**
  * Daffodil Validator implementation for ISO schematron
+ *
+ * Configuration
+ * - schematron.path=uri_string_to_schematron_file
+ * - schematron.svrl.file=uri_string_to_output_file
+ * - daffodil.rootSchema=schema_file_uri_string
```

Comment:
This needs additional documentation, something svrl.file is optional, and if provided a schematron report will be written there. If schematron.path is defined it will use that and expect it to be a .sch file. If that's not defined, it will use daffodil.rootSchema and attempt to extra embedded schematron rules.

Also, should `schematron.path` be changed to `schematron`? So that if you do something like this will work?

```scala
daffodil parse -s ... --validate schematron=path/to/schematron.sch
```

or is schematron complex enough that it really requires a config file?